lms/static/styles/frontend_apps.scss

```diff
@@ -31,6 +32,12 @@
 // Shared styles from frontend-shared package
 @use '@hypothesis/frontend-shared/styles';
 
+// Make the shared package utility styles available for
+// initial prototyping purposes.
+// NB: These styles have not been made explicitly public by this package
+// yet, so this inclusion may be temporary.
+@use '@hypothesis/frontend-shared/styles/util' as hyp-u;
```

Comment:
Does the `hyp-u` prefix do anything here? I thought that only mattered when you were referencing mixins/styles from within SASS. ie. I was under the belief that it doesn't alter the class names output by a module.

Reply:
We can't `@use` the shared util classes without namespacing because we are already `@use`-ing another module called `utils` in this module (the application's local utility classes). Since I had to alias to _something_, I figured `hyp-u` might help draw the connection between this module's "import" and the classes it contains/provides. Open to other approaches, and/or I could add a comment to help explain.

Reply:
> We can't @use the shared util classes without namespacing because we are already @use-ing another module called utils in this module (the application's local utility classes). Since I had to alias to something, I figured hyp-u might help draw the connection between this module's "import" and the classes it contains/provides. 

Ah, I see. Technically the existing module is called `utils` as opposed to util so there isn't a conflict. Maybe something like `<package name>-<module name>` could be used as a convention for third-party imports, where the package name could potentially be abbreviated if it is too verbose (eg. `fs-util` or something).